src/Github/CreateReleaseText.php

```diff
@@ -13,15 +14,14 @@ interface CreateReleaseText
 {
     /**
      * @psalm-param non-empty-string $repositoryDirectory
-     * @psalm-return non-empty-string
      */
     public function __invoke(
         Milestone $milestone,
         RepositoryName $repositoryName,
         SemVerVersion $semVerVersion,
         BranchName $sourceBranch,
         string $repositoryDirectory
-    ): string;
+    ): ChangelogReleaseNotes;
```

Comment:
BC Break 😱 Technically then this feature should go to v2.0

Reply:
This package has `"type": "application"` and is not published to packagist either - we can break internals as much as we want (and feel free to do so!)